Capabilities and limitation to Public Cloud Databases

Pirionfr opened the pull request:
add Capabilities and limitation to Public Cloud Databases

Commit:
add Capabilities and limitation to Public Cloud Databases

thild42 commented on pages/platform/databases/information_05_capabilities/guide.en-gb.md:
```diff
@@ -0,0 +1,105 @@
+slug: capabilities
+excerpt: Discover the capabilities and limitations of Public Cloud Databases
+section: General information
+order: 040
```
Bump to 050 as 040 is for information_04_faq

thild42 commented on pages/platform/databases/information_05_capabilities/guide.pt-pt.md:
```diff
@@ -0,0 +1,105 @@
+We currently offer seven database engines:
+
+### [MySQL](/publiccloud/databases/mysql/capabilities/)
+MySQL is an open-source fully-managed relational database service. It's fast, reliable, scalable, and user-friendly
```
Missing the ending "."

thild42 commented on pages/platform/databases/information_05_capabilities/guide.en-gb.md:
```diff
@@ -0,0 +1,105 @@
+We currently offer seven database engines:
+
+### [MySQL](/publiccloud/databases/mysql/capabilities/)
+MySQL is an open-source fully-managed relational database service. It's fast, reliable, scalable, and user-friendly
```
Missing the ending "."

thild42 commented on pages/platform/databases/information_05_capabilities/guide.en-gb.md:
```diff
@@ -0,0 +1,105 @@
+MySQL is an open-source fully-managed relational database service. It's fast, reliable, scalable, and user-friendly
+
+### [PostgreSQL](/publiccloud/databases/postgresql/capabilities/)
+PostgreSQL is an open-source fully managed and hosted relational database service. It's a high-performance engine that offers flexibility through variety of included advanced extensions 
```
Missing the ending "."

thild42 commented on pages/platform/databases/information_05_capabilities/guide.en-gb.md:
```diff
@@ -0,0 +1,105 @@
+PostgreSQL is an open-source fully managed and hosted relational database service. It's a high-performance engine that offers flexibility through variety of included advanced extensions 
+
+### [MongoDB](/publiccloud/databases/mongodb/capabilities/)
+MongoDB is fully managed document-oriented database management system scalable and does not require a predefined data schema
```
Missing the ending "."

thild42 commented on pages/platform/databases/information_05_capabilities/guide.en-gb.md:
```diff
@@ -0,0 +1,105 @@
+
+### Business
+The Business plan is more comprehensive and suitable for organizations with higher database demands.
+This is the first production-ready offering
```
Missing the ending "."

thild42 commented on pages/platform/databases/information_05_capabilities/guide.en-gb.md:
```diff
@@ -0,0 +1,105 @@
+
+### Essential
+The Essential plan typically offers the basic features and functionalities required for managing a database. 
+It is designed for tests, development environments and non-critical projects.
```
Suggestion: "It is designed for test and development environments or non-critical projects."

thild42 commented on pages/platform/databases/information_05_capabilities/guide.en-gb.md:
```diff
@@ -0,0 +1,105 @@
+
+(1) depend on the plan
+
+# Specific feature Plans
```
Should be "##" level.

thild42 commented on pages/platform/databases/information_05_capabilities/guide.en-gb.md:
```diff
@@ -0,0 +1,105 @@
+### Enterprise
+The enterprise pricing plan is the most advanced and robust option available for managed databases. 
+It is dedicated to organizations with complex and high-performance database requirements.
+
```
Maybe create a dedicated section like "### Plan capabilities overview" .

thild42 commented on pages/platform/databases/information_05_capabilities/guide.en-gb.md:
```diff
@@ -0,0 +1,105 @@
+---
+title: Capabilities and limitation
```
"Limitations" with a capital "L" and a trailing "s".

thild42 commented on pages/platform/databases/information_05_capabilities/guide.en-gb.md:
```diff
@@ -0,0 +1,105 @@
+Your choice of plan affects the number of nodes your cluster can run.
+
+
+## Last Updates
```
Add lastest update for secondary region.

OVH-Team-Guides committed:
Proofreading all

Deprecated fields removed from the header: section, order, slug
Only the updated: key is mandatory for date information.

OVH-Team-Guides committed:
Update index.md

OVH-Team-Guides commented:
Thank you for your contribution @Pirionfr
